src/Microsoft.DotNet.Interactive.Http.Parsing/Parsing/DynamicExpressionUtilities.cs

```diff
@@ -183,6 +184,13 @@ private static DateTimeOffset DefaultGetDateTimeOffset(bool isLocal)
                     else if (string.Equals(type.Value, "iso8601", StringComparison.OrdinalIgnoreCase))
                     {
                         format = "o";
+                        if (currentDateTimeOffset.Offset.TotalMinutes == 0)

```

Comment:
Do we have unit tests that cover each of the formatting cases covered in this function?